src/actions/interfaces.ts

```diff
@@ -8,6 +9,14 @@ export enum ActionType {
     Snippet = "SNIPPET"
 }
 
+// Don't change the types - used by bas-cli!
+export enum ActionJsonKey {
+    ActionType = "actionType",
+    CommandName = "commandName",
+    CommandParams = "commandParams",
+    Url = "url"
```

Comment:
I assume you mean Uri.

Reply:
I think url is correct.
Somebody here https://danielmiessler.com/study/difference-between-uri-url/#:~:text=The%20terms%20%E2%80%9CURI%E2%80%9D%20and%20%E2%80%9C,as%20HTTPs%20%2C%20FTP%20%2C%20etc. thinks the same.
But I can change.

Reply:
ok. we support opening URIs, so lets change it to uri.

Reply:
Are you sure?
Please see here: https://en.wikipedia.org/wiki/Uniform_Resource_Identifier:
Some URIs provide a means of locating and retrieving information resources on a network (either on the Internet or on another private network, such as a computer filesystem or an Intranet), these are Uniform Resource Locators (URLs). Other URIs provide only a unique name, without a means of locating or retrieving the resource or information about it, these are Uniform Resource Names (URNs). 
We support uri's which are url's but not urn's.